### PR TITLE
Remove redundant index on landing page

### PR DESCRIPTION
**Pull request by Bento007**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,12 +1,9 @@
 .. include:: ../README.rst
 
-
-Table of Contents
------------------
 .. toctree::
    :maxdepth: 5
+   :hidden:
```
Comment (calvinnhieu) [inline docs/index.rst]: What does this do?

**Comment by calvinnhieu** [comment]
What landing page is this for?

File: docs/index.rst (hunk 1 above)
Comment (Bento007) [reply docs/index.rst]: it's hides the TOC from the landing page, so it only shows up in the sidebar

**Comment by Bento007** [comment]
@calvinnhieu for the [HCA CLI read the docs](https://hca.readthedocs.io/en/latest/)

**Pull request merged**